CI/physmon/workflows/physmon_trackfitting_gsf.py

```diff
@@ -11,10 +11,19 @@
 
 setup = makeSetup()
 
+# Paths to pre-simulated data (from physmon_simulation_gsf.py)
+simDir = (setup.outdir / "../simulation_gsf/simulation").resolve()
+particlesPath = simDir / "particles_simulation.root"
+simhitsPath = simDir / "hits.root"
+
+# Verify simulation files exist
+assert particlesPath.exists(), f"Simulation not found: {particlesPath}"
+assert simhitsPath.exists(), f"SimHits not found: {simhitsPath}"
```

Comment:
after this the workflow can only be executed if the simulation script is run before right?

Reply:
Hmm yeah thats true. But I'm not sure I would make this optional out of the box. Changing it is trivial in principle though (removing the asserts, pass `None`s to the function).
Would you propose to make it optional / adaptive?